.github/ISSUE_TEMPLATE/00-bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
```

Comment:
The Issue template files should not be added. Otherwise, people will do the wrong things with bugs here.

Reply:
removed